CMakeLists.txt

```
@@ -1159,6 +1159,70 @@ ExternalProject_Add( fmt
 
 list(APPEND build_list fmt)
 
+
+################################
+# Fesapi
+################################
+
+if( ENABLE_VTK AND ENABLE_FESAPI )
```

Though I understand the intent to save build time, I would maybe not encode it here that VTK is currently required for FESAPI to be usable in the main code. There's no formal dependency in the build process, and it's something that GEOSX configuration should handle (and probably does in your GEOSX branch), but as far as building TPLs those could be enabled independently.
```suggestion
if( ENABLE_FESAPI )
```

---

Ok I agree